cumulusci/salesforce_api/package_install.py

```diff
@@ -86,6 +89,8 @@ def from_task_options(task_options: dict) -> "PackageInstallOptions":
                 )
             if "upgrade_type" in task_options:
                 options.upgrade_type = UpgradeType(task_options["upgrade_type"])
+            if "skip_handlers" in task_options:
+                options.skip_handlers = SkipHandlers(task_options["skip_handlers"])
```

Comment:
💭: Implementation note: The [`sf` flag](https://github.com/salesforcecli/plugin-packaging/blob/2421e1577f2168e4875a46361eaa2ae043b3a75c/src/commands/package/install.ts#L106-L113) allows multiple values and combines them into a [comma-separated string](https://github.com/salesforcecli/plugin-packaging/blob/2421e1577f2168e4875a46361eaa2ae043b3a75c/src/commands/package/install.ts#L140) for `PackageInstallCreateRequest`. We don't need to do this here now, but we might need to if they add more `SkipHandlers` in the future.

Reply:
Added a comment for future consideration.